test/views/declaration/spec/ViewMatchers.scala

```diff
@@ -131,6 +137,15 @@ trait ViewMatchers {
       )
   }
 
+  class ContainAnchorElementWithClassMatcher(name: String, value : String) extends Matcher[Element] {
```

Comment:
Doesnt look like you're using `name` in your lookup? just in your message? 

Also `value` is probably better named `href`

Reply:
I have addressed the naming convention, but the call here does not apply, because it is a string anchor that we are testing